routes/web.php

```diff
@@ -16,10 +16,12 @@
 */
 
 Route::get('/', function () {
-    return view('welcome')->name('home');
+    return view('welcome')->name('welcome ');
 });
 
 Route::get('/login', [AuthController::class, 'showLoginForm'])->name('login');
+Route::post('login', [AuthController::class, 'authenticate']);
 Route::get('/register', [AuthController::class, 'showRegistrationForm'])->name('register');
 Route::post('/register', [RegistrationController::class, 'register'])->name('register');
```

Comment:
named routes should be unique. Having 2 `register` routes is having an issue. Try submitting the register form